### PR TITLE
Expand beian options for users in mainland China

### PR DESCRIPTION
**Pull request by MartinMa94**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [X] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [X] Tests for the changes was maked (for bug fixes / features).
   - [X] Muse | Mist have been tested.
   - [X] Pisces | Gemini have been tested.
- [x] [Docs](https://github.com/theme-next/theme-next.org/tree/source/source/docs) in [NexT website](https://theme-next.org/docs/) have been added / updated (for features).

## PR Type
**What kind of change does this PR introduce?**

- [ ] Bugfix.
- [X] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

No feature of the public security record.

Issue resolved: N/A

## What is the new behavior?

For Chinese users, except for the ICP record, they need public security record, too. I added this feature in this pull request.

- Screenshots with this changes: 

- Link to demo site with this changes: [https://www.matalking.com/](https://www.matalking.com/)

### How to use?
In NexT `_config.yml`:
```yml
footer:
  beian:
    enable: true
    icp: 京ICP备19031359号-2
    gongan_id: 11010502038639
    gongan_num: 京公网安备 11010502038639号
    gongan_icon_url: /uploads/beian.png
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [X] No.

### CODE REVIEW

**Comment by 1v9** [comment]
原有的备案选项扩充即可，不用新增选项。

**Comment by MartinMa94** [comment]
> 原有的备案选项扩充即可，不用新增选项。

好的。另外这个 Mergeable 说我 LABEL 和 MILESTONE 没通过验证。这两个东西怎么加？

**Comment by MartinMa94** [comment]
> 原有的备案选项扩充即可，不用新增选项。

It has been modified.

File: _config.yml [hunk 1]
```diff
@@ -80,10 +80,18 @@ footer:
     # Version info of NexT after scheme info (vX.X.X).
     version: true
 
-  # Beian ICP information for Chinese users. See: http://www.beian.miit.gov.cn
+  # Beian ICP and gongan information for Chinese users.
+  # See: http://www.beian.miit.gov.cn
```
Comment (stevenjoezhang) [inline _config.yml]: # See: http://www.beian.miit.gov.cn, http://www.beian.gov.cn

File: _config.yml [hunk 2]
```diff
@@ -80,10 +80,18 @@ footer:
...
+  # See: http://www.beian.gov.cn
   beian:
     enable: false
     icp:
+    # The digit in th num of gongan beian.
```
Comment (stevenjoezhang) [inline _config.yml]: Do you mean `the`

File: layout/_partials/footer.swig [hunk 3]
```diff
@@ -1,7 +1,11 @@
 <div class="copyright">{#
 #}{%- set current = date(Date.now(), "YYYY") %}{#
 #}{%- if theme.footer.beian.enable %}{#
-#}  {{ next_url('http://www.beian.miit.gov.cn', theme.footer.beian.icp + ' ') }}{#
+#}{{ next_url('http://www.beian.miit.gov.cn', theme.footer.beian.icp + ' ') }}
+  {%- if theme.footer.beian.gongan_icon_url %}<span><img src="{{ theme.footer.beian.gongan_icon_url }}" style="display:inline-block;"/></span>
```
Comment (stevenjoezhang) [inline layout/_partials/footer.swig]: No need to use xhtml syntax `/>`

`style="display: inline-block;">` is OK

**Comment by stevenjoezhang** [comment]
需要在这个更新一下文档：https://github.com/theme-next/theme-next.org/pulls

File: _config.yml (hunk 2 above)
Comment (MartinMa94) [reply _config.yml]: Yeah. I typed wrongly. It should be the.

**Comment by MartinMa94** [comment]
> 需要在这里更新一下文档：https://github.com/theme-next/theme-next.org/pulls
> 
> >  GitHub**[theme-next/theme-next.org](https://github.com/theme-next/theme-next.org)**The website for NexT theme. Contribute to theme-next/theme-next.org development by creating an account on GitHub.

请问文档需要改什么地方？

**Comment by MartinMa94** [comment]
> 需要在这里更新一下文档：https://github.com/theme-next/theme-next.org/pulls
> 
> >  GitHub**[theme-next/theme-next.org](https://github.com/theme-next/theme-next.org)**The website for NexT theme. Contribute to theme-next/theme-next.org development by creating an account on GitHub.

目前文档中好像没有关于大陆用户备案的使用说明。
https://theme-next.org/docs/theme-settings/footer
应该在这个文档里，但是目前没有。

**Comment by 1v9** [comment]
你可以补上

**Comment by 1v9** [comment]
照着 Site Copyright Name 简单陈述即可。

**Comment by MartinMa94** [comment]
> 照着 Site Copyright Name 简单陈述即可。

@1v9 已补

**Comment by 1v9** [comment]
@all-contributors please add @MartinMa94 for code, doc

**Pull request merged**